docs/getting_started.md

```diff
@@ -46,8 +46,10 @@ cp ocs-ci.yaml.template ~/.ocs-ci.yaml
 In order to deploy a cluster to AWS with the Openshift Installer,
 you will need to download the pull secret for your account.
 Download this file from [openshift.com](https://cloud.openshift.com/clusters/install)
-and place in the `data` directory at the root level of the project.
-If there is no `data` directory, create one.
+and place in the `data` directory at the root level of the project
+(If there is no `data` directory, create one) or you can place the file in
+`$HOME/ocsci-data/`. Placing the data in `$HOME/ocsci-data/` is safer and you dont
```

Comment:
please try to also keep the line number under 79 characters, I see here in github that the word `dont` is on new line. BTW don't . 

The same line length below.

Reply:
we have let docs go upto 120 lines since its rendered in browser to keep it consistent with previous changes or else my line would appear truncated compared to existing doc.